### DISCUSSION TITLE
How to write my readme file awesome?

### DISCUSSION THREAD

**Discussion by massimo1220** [community/community#33553] [Other Feature Feedback, Questions, & Ideas] [answered]
How to write my readme file awesome?
If anyone knows the answer, pleas help me.

Thanks.Devstar0202

**Comment by MinJerry1129** [comment, marked as answer]
hello, @DevStar0202 

I wrote my readme file.
please look at my github profile.

@minjerry1129

**Reply by massimo1220** [reply]
@MinJerry1129

Thanks.

**Comment by Sagar-Sharma-7** [comment]
Hello, brother
You can check out my GitHub profile.
